sdk/utils/endpoints.js

```diff
@@ -10,4 +10,13 @@ const loginEndpoint = () => {
 const userInfoEndpoint = accessToken =>
   `https://auth-testing.iduruguay.gub.uy/oidc/v1/userinfo?authorization=${accessToken}`;
 
-export { loginEndpoint, userInfoEndpoint, tokenEndpoint };
+const logoutEndpoint = () => {
+  const { idToken, postLogoutRedirectUri, state } = getParameters();
+  return (
+    idToken &&
+    postLogoutRedirectUri &&
+    `https://auth-testing.iduruguay.gub.uy/oidc/v1/logout?id_token_hint=${idToken}&post_logout_redirect_uri=${postLogoutRedirectUri}&state=${state}`
+  );
```

Comment:
```suggestion
  return `https://auth-testing.iduruguay.gub.uy/oidc/v1/logout?id_token_hint=${idToken}&post_logout_redirect_uri=${postLogoutRedirectUri}&state=${state}`;
```
Este chequeo es redundante dado que ya lo haces en la línea 40 del archivo `logout.js` en la carpeta `requests`.